src/Presentation/ListFacetHtmlBuilder.php

```diff
@@ -51,6 +70,7 @@ private function buildCheckboxesViewModel( FacetConfig $config, PropertyConstrai
 				'label' => $valueCount->value,
 				'count' => $valueCount->count, // FIXME: count is now showing in the UI for some reason
 				'checked' => in_array( $valueCount->value, $selectedValues ), // TODO: test with multiple types of values
+				'value-id' => $valueCount->value,
```

Comment:
This does not make sense

I guess you put this here because for values of type `Item ID` we want to show the associated Wikibase Item label instead of the ID itself. In those cases we could theoretically provide the label to the label key here, and the value itself to `value-id`. Values of other types do not have an ID, so it does not work for them.

There also is uncertainty about where we want to get the label for item ids. We could do this on the frontend and get ourselves form progressive enhancement. 

For now, just ingore the item ID complications. What we need in the UI is the value itself, since they is what we need in the query string. We can decide how to show labels instead of IDs later.

Reply:
The attribute is only intended to be consumed by the JS.

I am not sure what would be the correct term for the value that we use for the `haswbfacet` query, so I put `value-id` for now as it is an identifier for the value. Is there a more suitable term that can be used for the data attribute?

Reply:
`value`. I suppose renaming to that would be OK for now, though we might end up only needing `value` and not `label`